<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoronInjector</class>
 <widget class="QDialog" name="BoronInjector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="HeaterTemp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Heater Temp</string>
   </property>
   <widget class="QProgressBar" name="BoronHeaterBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
progressbar::chunk: 10px;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BoronTempSetPoint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="BoronTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Temp</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Done">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="OnOff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="title">
    <string>On / Off</string>
   </property>
   <widget class="QPushButton" name="On">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="HeaterCharge">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>91</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Heater Charge</string>
   </property>
   <widget class="QProgressBar" name="BoronChargeBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
progressbar::chunk: 10px;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BoronChargeSetpoint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="BoronCharge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Charge</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Done</sender>
   <signal>clicked()</signal>
   <receiver>BoronInjector</receiver>
   <slot>Done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoronTempSetPoint</sender>
   <signal>sliderReleased()</signal>
   <receiver>BoronInjector</receiver>
   <slot>SetBoronHeaterSetPoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>On</sender>
   <signal>clicked()</signal>
   <receiver>BoronInjector</receiver>
   <slot>SetBoronHeaterOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetBoronHeaterSetPoint()</slot>
  <slot>Done()</slot>
  <slot>SetBoronHeaterOn()</slot>
  <slot>SetBoronChargeSetPoint()</slot>
 </slots>
</ui>
